deid/dicom/groups.py

```diff
@@ -55,7 +56,10 @@ def extract_values_list(dicom, actions, fields=None):
         if action["action"] == "FIELD":
             for uid, field in subset.items():
                 if field.element.value not in ["", None]:
-                    values.add(field.element.value)
+                    if type(field.element.value) is MultiValue:
```

Comment:
Let's use isinstance here, it's a bit faster: https://switowski.com/blog/type-vs-isinstance
```suggestion
                    if isinstance(field.element.value, MultiValue):
```